wasm/jit/jit_amd64_test.go

```diff
@@ -1402,6 +1402,97 @@ func TestAmd64Builder_handleCall(t *testing.T) {
 	})
 }
 
+func TestAmd64Builder_handleLoad(t *testing.T) {
+	for i, tp := range []wazeroir.SignLessType{
+		wazeroir.SignLessTypeI32,
+		wazeroir.SignLessTypeI64,
+		wazeroir.SignLessTypeF32,
+		wazeroir.SignLessTypeF64,
+	} {
+		tp := tp
+		t.Run(fmt.Sprintf("%d", i), func(t *testing.T) {
+			eng := newEngine()
+			builder := requireNewBuilder(t)
+			builder.initializeReservedRegisters()
+			// Before load operations, we must push the base offset value.
```

Comment:
is 100 a magic number? what determines the base? if arbitrary, finish with "... for this test, we use an arbitrary value"